.github/ISSUE_TEMPLATE/frontend_report.yml

```
@@ -0,0 +1,75 @@
+name: "🌐 Frontend (Issue / Optimization / Documentation)"
+description: Report an issue specifically related to the website frontend. For other inquiries, please use the Discussions section.
```

```suggestion
description: Report an issue specifically related to the frontend. For other inquiries, please use the Discussions section.
```
Generally for this PR we should decide between `frontend` or `website`. Personally I find `website` more user friendly since not everyone know what a frontend is